src/test/java/seedu/address/model/person/DisplayClientTest.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.TypicalPersons.ALICE;
+import static seedu.address.testutil.TypicalPersons.BENSON;
+
+import org.junit.jupiter.api.Test;
+
+public class DisplayClientTest {
+
```

Comment:
Is it possible for displayClientNull and displayClientNonNull to be extracted out of the functions and be used as a common test objects?
```suggestion
DisplayClient displayClientNull = new DisplayClient(null);
DisplayClient displayClientNonNull = new DisplayClient(ALICE);
```

Reply:
I think that if it is not modified anywhere within the method it would be cleaner to just use a common test object. What do you think?